docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md

```diff
@@ -0,0 +1,727 @@
+---
```

Comment:
General comment: please check the licenses for your pictures and include relevant info - please refer to the TRG 7.07

Reply:
Licenses have been checked by the team